viral/cache_2p_sessions.py

```diff
@@ -411,60 +499,68 @@ def main() -> None:
 
     # for mouse_name in ["JB017", "JB019", "JB020", "JB021", "JB022", "JB023"]:
     redo = True
-    for mouse_name in ["JB031"]:
+    for mouse_name in ["JB032"]:
         metadata = gsheet2df(SPREADSHEET_ID, mouse_name, 1)
         for _, row in metadata.iterrows():
             try:
                 print(f"the type is {row['Type']}")
 
                 date = row["Date"]
                 session_type = row["Type"].lower()
-
-                if (
-                    not redo
-                    and (
-                        HERE.parent / "data" / "cached_2p" / f"{mouse_name}_{date}.json"
-                    ).exists()
-                ):
-                    print(f"Skipping {mouse_name} {date} as already exists")
-                    continue
-
-                if (
-                    "learning day" not in session_type
-                    and "reversal learning" not in session_type
-                ):
-                    print(f"Skipping {mouse_name} {date} {session_type}")
-                    continue
-
-                if not row["Sync file"]:
-                    print(
-                        f"Skipping {mouse_name} {date} {session_type} as no sync file"
+                wheel_blocked = row["Wheel blocked?"].lower() == "yes"
```

Comment:
I think this will error on old mice where there's no wheel blocked column, I would check if the column exists